src/pages/code-for-good.js

```diff
@@ -0,0 +1,317 @@
+                                All you need is a{' '}
+                                <StyledParagraphLink href="https://cloud.mongodb.com">
+                                    free Atlas account
+                                </StyledParagraphLink>{' '}
+                                and{' '}
+                                <StyledParagraphLink href="https://docs.mongodb.com/manual/installation/#mongodb-community-edition-installation-tutorials">
+                                    MongoDB Community Edition{' '}
+                                </StyledParagraphLink>{' '}
+                                to build your own O-FISH instance.
+                            </BodyText>
+
+                            <StyledLink href="https://o-fish.github.io">
+                                Start building now
+                            </StyledLink>
+                        </VesselAppTextContainer>
+
+                        <VesselAppTextContainer>
+                            <ArticleH2>
+                                Contribute code to help protect our oceans
+                            </ArticleH2>
+                            <BodyText>
+                                Submit new features, translations, and styles to
+                                O-FISH. This is an open source application;
+                                anyone can contribute.
+                            </BodyText>
+
+                            <StyledLink href="#">
+                                Contribute your code
+                            </StyledLink>
+                        </VesselAppTextContainer>
+
+                        <VesselAppTextContainer>
+                            <ArticleH2>
+                                Collaborate with MongoDB Developers around the
+                                world
+                            </ArticleH2>
+                            <BodyText>
+                                Connect with your fellow MongoDB developers to
+                                discuss how the O-FISH app works and how to make
+                                it better.
+                            </BodyText>
+                            <StyledLink href="#">
+                                Discuss O-FISH with the community
+                            </StyledLink>
+                        </VesselAppTextContainer>
+                    </div>
+
+                    <VesselImageDiv>
+                        <StyledVesselImage
+                            src={VesselRecord}
+                            alt="Sample O-Fish application with vessel boarding information"
+                        />
+                        <VesselCaptionText>
+                            Realm Sync enables officers to see and submit
+                            boarding reports through the mobile app even when
+                            they are out at sea, offline.
+                        </VesselCaptionText>
```

Comment:
Thanks for looking at this! Let's make this `figcaption` child a `P3` like we do in articles to distinguish this text from the text below it (and play around with the color a bit). I think the design should follow our standards for `figure` from articles. One such article you can see this in is /quickstart/java-setup-crud-operations

This is what we do in articles:
```
const StyledCaption = styled(P3)`
    color: ${colorMap.greyLightTwo};
`;
const Caption = StyledCaption.withComponent('figcaption');
```